Utilities/priority_queue.py

```diff
@@ -12,6 +12,8 @@ class PriorityQueue():
         The priority queue.
     """
 
+    count = 900000
```

Comment:
don't hardcode configurable values, use constants.
```suggestion
    count = DEFAULT_COUNT
```